error.vue

```diff
@@ -39,20 +28,21 @@ async function reload() {
         <span timeline-title-style>Error</span>
       </template>
       <slot>
-        <form p5 grid gap-y-4 @submit="reload">
-          <div text-lg>
-            Something went wrong
+        <div p5 grid gap-y-4>
+          <div text-xl font-bold>
+            {{ message }}
           </div>
           <div text-secondary>
-            {{ message }}
+            {{ subtitle }}
           </div>
-          <button flex items-center gap-2 justify-center btn-solid text-center :disabled="state === 'reloading'">
-            <span v-if="state === 'reloading'" block animate-spin preserve-3d>
-              <span block i-ri:loader-2-fill />
-            </span>
-            {{ state === 'reloading' ? 'Reloading' : 'Reload' }}
-          </button>
-        </form>
+          <NuxtLink
+            :aria-label="$t('nav.back')"
+            class="btn-text inline-flex flex-items-center -ml-5"
+            @click="$router.go(-1)"
+          >
+            <span i-ri:arrow-left-line /> {{ $t('nav.back') }}
+          </NuxtLink>
```

Comment:
Just to be clear. We are dropping the reload in favor of a back button?  I didn't see that in the ticket so wanted to make sure we update the ticket prior to QA if this came from an external conversation.

Also on a 500 error, we may not want them to go back right?